BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeHttpClient.kt

```diff
@@ -124,6 +125,7 @@ internal class BraintreeHttpClient(
         if (authorization is TokenizationKey) {
             request.addHeader(CLIENT_KEY_HEADER, authorization.bearer)
         }
+        authorization?.bearer?.let { token -> request.addHeader("Authorization", "Bearer $token") }
```

Comment:
Can we have a test to cover this? To make sure it is only added in the cases we expect?

Reply:
I added a few tests covering this logic.